aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+import { Box } from 'grommet';
+import { ActivitiesNavigationalCards } from '../card';
+
+const skeleton = { message: { start: 'Loading', end: 'Content Loaded' } };
+
+export const DashBoardSkeletonExample = () => {
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => setLoading(!loading), 3000);
+  }, [loading]);
+
+  return (
+    <>
+      <Box width="large" skeleton={loading ? skeleton : undefined}>
```

Comment:
I noticed we don't have any animation on this. Is that going to be handled in a separate PR/next step?

Reply:
yeah I left that out while Kenny is looking more into what we want for guidance we can have it as a next step

Reply:
Perfect, makes sense.